### PR TITLE
Fixed interaction of trail points with ATM.

### PR DESCRIPTION
**Pull request by jccosta**

### CODE REVIEW

File: modules/base/shaders/renderabletrail_fs.glsl [hunk 1]
```diff
@@ -45,18 +46,31 @@ Fragment getFragment() {
     frag.depth = vs_positionScreenSpace.w;
     frag.blend = BLEND_MODE_ADDITIVE;
 
+    vec4 depthCorrection = vec4(0.0, 0.0, 100.0, 0.0);
```
Comment (alexanderbock) [inline modules/base/shaders/renderabletrail_fs.glsl]: This can be removed?

File: modules/base/shaders/renderabletrail_fs.glsl [hunk 2]
```diff
@@ -45,18 +46,31 @@ Fragment getFragment() {
...
+
+        frag.color.a = transparencyCorrection;
+    }    
+
 
     // G-Buffer
-    frag.gPosition  = vs_gPosition;
+    //frag.gPosition  = vs_gPosition;
+    // JCC: We need to fix the precision here.
+    frag.gPosition = vs_gPosition + vec4(0.0, 0.0, 100.0, 0.0);
```
Comment (alexanderbock) [inline modules/base/shaders/renderabletrail_fs.glsl]: A bit more of a comment why we need the 100.0 here

File: modules/base/shaders/renderabletrail_fs.glsl [hunk 3]
```diff
@@ -45,18 +46,31 @@ Fragment getFragment() {
...
+
     // There is no normal here
     // TODO: Add the correct normal if necessary (JCC)
```
Comment (alexanderbock) [inline modules/base/shaders/renderabletrail_fs.glsl]: TODO comment can be deleted?

File: modules/base/shaders/renderabletrail_fs.glsl (hunk 1 above)
Comment (jccosta) [reply modules/base/shaders/renderabletrail_fs.glsl]: Fixed.

File: modules/base/shaders/renderabletrail_fs.glsl (hunk 2 above)
Comment (jccosta) [reply modules/base/shaders/renderabletrail_fs.glsl]: Done.

File: modules/base/shaders/renderabletrail_fs.glsl (hunk 3 above)
Comment (jccosta) [reply modules/base/shaders/renderabletrail_fs.glsl]: Done.

**Pull request merged**